apps/server/src/modules/learnroom/controller/dto/card/card.response.ts

```diff
@@ -0,0 +1,44 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { DecodeHtmlEntities } from '@shared/controller';
+import { VisibilitySettingsResponse } from './visibility-settings.response';
+
+export class ContentElementResponse {}
+
+type BoardCardType = 'task' | 'content' | 'legacy-task' | 'legacy-lesson';
+
+export class CardResponse {
```

Comment:
this could be an interface instead, and individual cards should be separated modules
in that case, TaskCardResponse should implent the interface too
task-card/controller/dto/task-card.response.ts

Reply:
will be discussed latter